service/src/main/resources/config/verily/sdd/original/ui/top_level.json

```diff
@@ -324,7 +334,7 @@
       "id": "tanagra-snp",
       "title": "SNP variant",
       "category": "Program data",
-      "columns": [{ "key": "id", "width": "100%", "title": "SNP variant" }],
+      "columns": [{ "key": "name", "width": "100%", "title": "SNP variant" }],
```

Comment:
I haven't updated the `snp` entity in the `sdd` underlays yet, only the `snp_refresh0323` one, so I think this will fail.

Reply:
Right, reverted.